custom-logger/src/worker.ts

```diff
@@ -0,0 +1,54 @@
+import { DefaultLogger, makeTelemetryFilterString, Runtime, Worker } from '@temporalio/worker';
+import * as activities from './activities';
+import { createLogger } from './logging';
+
+const winstonLogger = createLogger({
+  isProduction: process.env.NODE_ENV === 'production',
+  logFilePath: process.env.WORKER_LOG_PATH || '/var/log/worker.log',
+});
+
+async function main() {
+  // @@@SNIPSTART typescript-core-telemetry-options
+  // Configure Rust Core runtime to export SDK logs
+  Runtime.install({
+    // Install a logger to collect logs generated by Node.js Workers and Rust Core.
+    logger: new DefaultLogger('DEBUG', (entry) => {
+      //
+      winstonLogger.log({
+        level: entry.level.toLowerCase(),
+        message: entry.message,
+        timestamp: Number(entry.timestampNanos / 1_000_000n),
+        ...entry.meta,
+      });
+    }),
+    // Telemetry options control how logs are exported out of Rust Core.
+    telemetryOptions: {
+      logging: {
+        // By default, Core logs go directly to console. Setting the `forward` property here (to an
+        // empty object) enables forwarding of logs from Rust Core to the Node.js logger.
+        forward: {},
+        // This filter determines which logs should be forwarded from Rust Core to the Node.js logger.
```

Comment:
I would add a note that you would likely not want this to be set to DEBUG in a production setting. Probably WARN is enough (and in the line above).